### PR TITLE
Updates for new unity SDK.

### PR DESCRIPTION
**Pull request by scottdangelo**
Update the SpeechSandboxStreaming Script for changes.
Remove references to uname and passwd.
Change to using Assistant V2 api.
Change to use Assistant Skills.

### CODE REVIEW

File: SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs [hunk 1]
```diff
@@ -120,23 +104,18 @@ public class SpeechSandboxStreaming : MonoBehaviour
         }
         else
         {
-            throw new WatsonException("Please provide either username and password or IAM apikey to authenticate the service.");
+            throw new IBMException("Plesae provide IAM ApiKey for the Speach To Text service.");
```
Comment (markstur) [inline SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs]: s/Plesae/Please/

Comment (markstur) [reply SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs]: s/Speach/Speech/

File: SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs [hunk 2]
```diff
@@ -146,16 +125,23 @@ public class SpeechSandboxStreaming : MonoBehaviour
         }
         else
         {
-            throw new WatsonException("Please provide either username and password or IAM apikey to authenticate the service.");
+            throw new IBMException("Plesae provide IAM ApiKey for the Watson Assistant service.");
```
Comment (markstur) [inline SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs]: Plesae

File: SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs [hunk 3]
```diff
@@ -295,83 +285,87 @@ private void OnRecognize(SpeechRecognitionEvent result, Dictionary<string, objec
...
+                        var input = new MessageInput()
+                        {
+                            Text = transcript_text,
+                            Options = new MessageInputOptions()
+                            {
+                                ReturnContext = false
+                            }
+                        };
+                        Debug.Log("Input to Assistant:" + input.Text);
+                        //_assistant.Message(OnMessage, assistantId, sessionId, input);
```
Comment (markstur) [inline SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs]: delete commented out and redundant line

File: SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs [hunk 4]
```diff
@@ -295,83 +285,87 @@ private void OnRecognize(SpeechRecognitionEvent result, Dictionary<string, objec
...
-                foreach (RuntimeEntity entity in messageResponse.entities)
+                foreach (RuntimeEntity entity in resp.Result.Output.Entities)
                 {
-                    Debug.Log("entityType: " + entity.entity + " , value: " + entity.value);
-                    if (entity.entity == "material")
-                    {
-                        currentMat = entity.value;
+                    Debug.Log("entityType: " + entity.Entity + " , value: " + entity.Value);
+                    if (entity.Entity == "material")
+                    {
```
Comment (markstur) [inline SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs]: there's a whitespace and indent issue here

File: SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs [hunk 5]
```diff
@@ -35,21 +35,12 @@ public class SpeechSandboxStreaming : MonoBehaviour
     public List<AudioClip> helpClips;
 
     [SerializeField]
-    private fsSerializer _serializer = new fsSerializer();
+   //private fsSerializer _serializer = new fsSerializer();
```
Comment (markstur) [inline SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs]: delete commented out line

File: SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs (hunk 1 above)
Comment (scottdangelo) [reply SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs]: Plesae is Latin, you Cretan!

File: SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs (hunk 4 above)
Comment (scottdangelo) [reply SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs]: I don't see the issue.

File: SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs (hunk 1 above)
Comment (scottdangelo) [reply SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs]: done

File: SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs (hunk 3 above)
Comment (scottdangelo) [reply SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs]: done

File: SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs (hunk 5 above)
Comment (scottdangelo) [reply SpeechSandbox/Assets/Scripts/SpeechSandboxStreaming.cs]: done

**Pull request merged**